GoedeDoelenHelpen/ClientApp/src/app/home/partials/home-functions/home-functions.component.html

```diff
@@ -11,7 +11,7 @@
           <li (click)="setPhoneImage('dashboard_v3.svg', 3)" [class.active]="selected == 3">
             Houdt je voortgang bij en vergroot je draagvlak via het dashboard</li>
         </ul>
-        <a appRoundedButton mat-raised-button color="accent">
+        <a (click)="toRegistration()" appRoundedButton mat-raised-button color="accent">
```

Comment:
Niet echt gewenste manier voor simple navigatie `routerLink` zou beter zijn en heb ik voor deze knop geloof ik al gedaan in mijn branch